Sources_v3/Controllers/ListDatabaseObserver.swift

```diff
@@ -200,3 +201,24 @@ extension ListDatabaseObserver where DTO == Item {
         )
     }
 }
+
+public extension UITableView {
+    func applyListChanges<T: Equatable>(changes: [ListChange<T>]) {
+        beginUpdates()
+        
+        for change in changes {
+            switch change {
+            case let .insert(_, index: index):
+                insertRows(at: [index], with: .automatic)
+            case let .move(_, fromIndex: fromIndex, toIndex: toIndex):
+                moveRow(at: fromIndex, to: toIndex)
+            case let .update(_, index: index):
+                reloadRows(at: [index], with: .automatic)
+            case let .remove(_, index: index):
+                deleteRows(at: [index], with: .automatic)
+            }
+        }
+        
+        endUpdates()
+    }
+}
```

Comment:
I don't think we're ready to have this in the SDK yet. We'd need to make this properly - have the animations parametrized etc.. I think this can live in the sample app though :)